udp_driver/src/udp_bridge_node.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 the Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "udp_driver/udp_receiver_node.hpp"
+#include "udp_driver/udp_sender_node.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+#include <io_context/io_context.hpp>
+#include <lifecycle_msgs/msg/state.hpp>
+#include <lifecycle_msgs/msg/transition.hpp>
+
+#include <memory>
+
+using drivers::common::IoContext;
+using drivers::udp_driver::UdpReceiverNode;
+using drivers::udp_driver::UdpSenderNode;
+using lifecycle_msgs::msg::State;
+using lifecycle_msgs::msg::Transition;
+
+int main(int argc, char ** argv)
```

Comment:
might be good to explain the purpose of this `bridge` node with a short comment or something